content/docs/07-reference/04-stream-helpers/02-streaming-text-response.mdx

```diff
@@ -6,8 +6,8 @@ description: Learn to use StreamingTextResponse helper function in your applicat
 # `StreamingTextResponse`
 
 <Note type="warning">
-  This helper function is deprecated. Use `streamText.toDataStreamResponse()`
-  instead.
+  `StreamingTextResponse` has been removed in AI SDK 4.0. Use
+  `streamText.toDataStreamResponse()` instead.
```

Comment:
Linking to the reference would be helpful

https://sdk.vercel.ai/docs/reference/ai-sdk-core/stream-text#to-data-stream-response